social/src/test/java/org/exoplatform/platform/qa/ui/social/smoke/SOC_HomePageTestIT.java

```diff
@@ -0,0 +1,203 @@
+     * the number of likers is updated
+     */
+
+    // get the id of the webContent created
+    String id = $(byClassName("contentShare")).parent().getAttribute("id").split("ActivityContextBox")[1];
+
+    // click on the like button of the webContent
+    $(byXpath("//*[@id=\"LikeLink" + id + "\"]")).click();
+
+    $(byXpath("//*[@id=\"UnLikeLink" + id + "\"]")).waitUntil(Condition.appears, Configuration.timeout);
+    /*
+     * Step number: 2 Step Name: Check Likes part Step Description: - Check
+     * avatar - Mouse over the avatar Input Data: Expected Outcome: - Avatar of
+     * liker is added into likes part, the oldest liker is displayed at the
+     * right and the newest at the left. - Profile pictures of users popup
+     */
+    /*
+     * mouseOver(ELEMENT_PUBLICATION_WHOLIKED, true);
+     * mouseHoverByJavaScript(ELEMENT_PUBLICATION_WHOLIKED, 2);
+     * waitForAndGetElement(ELEMENT_PUBLICATION_WHOLIKEDPOPUP);
+     */
+    ELEMENT_WHO_LIKED_POPUP.waitUntil(Condition.appears, Configuration.timeout);
+    navigationToolbar.goToSiteExplorer();
+    siteExplorerHome.deleteData(name);
+
+  }
+
+  /**
+   * <li>Case ID:121909.</li>
+   * <li>Test Case Name: Add comment.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li>
+   * <li>Case ID:121933.</li>
+   * <li>Test Case Name: Delete comment.</li>
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li>
+   */
+  @Test
+  public void test02_15_AddDeleteComment() {
+    info("Test 2: Add comment");
+
+    String name = "name" + getRandomNumber();
+    String content = "content" + getRandomNumber();
+    String content2 = "content2" + getRandomNumber();
+
+    navigationToolbar.goToSiteExplorer();
+    siteExplorerHome.goToAddNewContent();
+    info("Create new file document");
+    createNewDocument.createNewDoc(CreateNewDocument.selectDocumentType.WEBCONTENT);
+    createNewDocument.addNewWebContent(name, content);
+    createNewDocument.saveAndClose();
+    Utils.pause(5000);
+    homePagePlatform.goToHomePage();
+
+    /*
+     * Step Number: 1 Step Name: Add comment for activity Step Description: - Go
+     * to Intranet home - Select the activity - Click comment icon to show input
+     * text field - input the comment and click comment Input Data: Expected
+     * Outcome: - Comment will be shown in comment section of activity
+     */
+
+    // get the id of webContent created
+    String id = $(byClassName("contentShare")).parent().getAttribute("id").split("ActivityContextBox")[1];
+    // click on comment icon
+    $(byXpath("//*[@id=\"CommentLink" + id + "\"]")).click();
+    // insert comment
+    $(byId("cke_CommentTextarea" + id)).waitUntil(Condition.appears, Configuration.timeout).click();
+    executeJavaScript("CKEDITOR.instances.CommentTextarea" + id + ".insertText(\"test\")", "");
+    // click on the button comment
+    $(byText("Comment")).waitUntil(Condition.enabled, Configuration.timeout).click();
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.